versions.config

```diff
@@ -1,6 +1,6 @@
 # config vars for the root file system
-HYPRIOT_OS_VERSION="v1.1.1"
-ROOTFS_TAR_CHECKSUM="5698b4f7363316d25ce735024836b499a4306802518b81d7231d457bbefd4f53"
+HYPRIOT_OS_VERSION="v1.2.0"
```

Comment:
@firecyberice could you please update to v1.2.1, just to get the very latest version in.